packages/round-manager/src/features/round/RoundApplicationForm.tsx

```diff
@@ -223,11 +220,17 @@ export function RoundApplicationForm(props: {
         operatorWallets: props.initialData.program.operatorWallets,
       } as Round;
 
-      await createRound({
-        roundMetadataWithProgramContractAddress,
-        applicationQuestions,
-        round,
-      });
+      /*TODO: detect version from round impl contract? */
+      const useNewRoundInit = chain.id === pgnTestnet.id;
```

Comment:
@vacekj we need to enable this for all the chains since Allo is deploying/upgrade the contracts everywhere